warehouse/ingest-core/src/test/java/datawave/ingest/mapreduce/EventMapperTest.java

```diff
@@ -15,30 +15,38 @@
 import org.apache.accumulo.core.data.Value;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.io.LongWritable;
-import org.apache.hadoop.mrunit.mapreduce.MapDriver;
+import org.apache.hadoop.mapreduce.Mapper;
+import org.easymock.EasyMockRule;
+import org.easymock.Mock;
+import org.junit.After;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
+import static org.easymock.EasyMock.*;
+import static org.junit.Assert.*;
 
 import java.io.IOException;
 import java.util.Map;
 
 public class EventMapperTest {
     
+    @Rule
+    public EasyMockRule easyMockRule = new EasyMockRule(this);
+    
+    @Mock
+    private Mapper.Context mapContext;
+    
     private Configuration conf;
-    private MapDriver<LongWritable,RawRecordContainer,BulkIngestKey,Value> driver;
```

Comment:
Wasn't expecting to see changes here of all places. Funny...the hoops we jump thru sometimes to beat dependencies into submission :) All in all though, was expecting a lot more pain to be inflicted by this upgrade. This is great

Reply:
Yeah, it seems mrunit was retired years ago. I forget exactly what tripped updating it-I think I got there through Arquillian updates. Overall, the end changes aren’t too bad. We are, however, completely ignoring wildfly’s new security system and using the old one—took a while to find that as the best option. But as long as it’s still there, we can keep upgrading...